pullrequest.js

```diff
@@ -77,6 +93,25 @@ chrome.storage.sync.get({url: ''}, function(items) {
 
         $body.on('click', '.js-collapse-deletions', collapseDeletions);
 
+        if (items.tabSwitchingEnabled) {
```

Comment:
So does this handle changing the setting without restarting the extension and all? If I'm reading it right, it does, because this is a callback on sync...?